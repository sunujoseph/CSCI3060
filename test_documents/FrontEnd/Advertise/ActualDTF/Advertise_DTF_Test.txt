03 Lenovo Tab M10            userSS          030000190
00 userSS          SS 222222.22
