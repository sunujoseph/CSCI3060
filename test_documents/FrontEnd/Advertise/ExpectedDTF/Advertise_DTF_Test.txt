03 Lenovo Tab M10      userSS          030 190.00
00 userSS          SS 222222.22