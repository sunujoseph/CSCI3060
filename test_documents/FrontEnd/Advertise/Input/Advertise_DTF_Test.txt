login
userSS
advertise
Lenovo Tab M10
190
30
logout
