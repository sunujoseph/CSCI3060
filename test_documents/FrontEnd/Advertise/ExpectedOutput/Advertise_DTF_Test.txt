Enter Username: 
Enter Item Name: 
Enter Minimum Bid: 
Enter Number Of Days Until Auction Ends: 